docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -106,33 +80,97 @@ spec:
       verbs: ['*']
 ```
 
-#### Combining Policies
+Because this sample policy requires that at least one user with the `auditor` role is present
+as a moderator to start SSH or Kubernetes sessions, a user assigned this `prod-access` role 
+won't be able to start any sessions on matching resources until the policy requirements are fulfilled.
+
+### Required fields
+
+The following are required fields for `require_session_join`:
+
+|Option|Type|Description|
+|---|---|---|
+|`name`|String|The name of the require policy|
+|`filter`|Filter|An expression that, if it evaluates to true for a given user, enables the user to be present in a moderated session.|
+|`kinds`|List|The kind of session—SSH, Kubernetes, or both—that the policy applies to.|
+|`modes`|List|The participant mode—observer, moderator, or peer—that the user joining the moderated session must match to satisfy the policy.|
```

Comment:
Since these are enums, I would list in code style the allowed values. Otherwise, a user has to guess them (e.g., `SSH` or `ssh`?).